### ISSUE TITLE
Search Ordering Query Param

### ISSUE THREAD

**Issue by enoziak** [omdbapi/OMDb-API#19]
It would be awesome to have the ability to order the search results by year. Below is an example of the issue i'm running into.

http://www.omdbapi.com/?t=mother!&y=2017 - this returns the Jennifer Lawrence movie that just came out. (total flop BTW)

http://www.omdbapi.com/?s=mother!&y=2017 - this has no mentioned of the new Jennifer Lawrence movie. If the search method did take into account date in the order and allowed for paging based on date and not relevance or however it's working right now that would be sweet. Maybe add an 'order' query param?

Let me know if there is a way to do this and i'm just missing it... :)

Thanks!

**Comment by josezulu** [comment]
If you do a search and sort only by year, you get undesired results listed first when in descending order. I tried this programmatically once, and I was getting "specials" and "extra documentaries" before the actual movie I was looking for. Try a search for the matrix, and you'll see all sorts of undesired results which year is later than all 3 movies.

Perhaps, a relevance + year order could work?

I think your suggestion of optional parameters is great.

**Comment by enoziak** [comment]
Agreed! A couple parameters would be helpful. Right now i'm running the title search and regular search and grouping the results. Throwing the title result to the top as it's probably the most likely and then dropping the weirder results below.
